pkg/cmd/cli/cmd/deploy.go

```diff
@@ -175,7 +200,9 @@ func (o DeployOptions) RunDeploy() error {
 			}
 			return list, nil
 		},
-
+		LogsForConfigFn: func(config *deployapi.DeploymentConfig, opts deployapi.DeploymentLogOptions) (io.ReadCloser, error) {
```

Comment:
Everything here looks suspiciously like an already existing client interface, I am just consistent with the existing design.